src/presentation/ui/component/form/selectInput.templ

```diff
@@ -88,3 +88,80 @@ templ SelectInputWithLabelValuePair(
 		<i class="ph-bold ph-caret-down pointer-events-none absolute right-2.5 top-3.5 text-sm"></i>
 	</div>
 }
+
+script MultipleSelectInputLocalState() {
+	document.addEventListener('alpine:init', () => {
+		Alpine.data('multipleSelectInput', () => ({
+			getFormattedSelectedItems(bindSelectedItemsPath) {
+				if (bindSelectedItemsPath.length == 0) {
+					return '--';
+				}
+
+				let formattedSelectedItems = bindSelectedItemsPath.join(', ');
+				if (formattedSelectedItems.length > 80) {
+					return formattedSelectedItems.substr(0, 80) + '...';
+				}
+				return formattedSelectedItems;
+			},
+			isOptionsExpanded: false,
+			closeDropdown() {
+				this.$refs.multipleSelectInputOptions.classList.add('hidden');
+				this.isOptionsExpanded = false;
+			},
+			toggleDropdownDisplay() {
+				if (this.isOptionsExpanded) {
+					this.closeDropdown();
+					return;
+				}
+
+				this.$refs.multipleSelectInputOptions.classList.remove('hidden');
+				this.isOptionsExpanded = true;
+			},
+		}));
+	});
+}
+
+templ MultipleSelectInput(
```

Comment:
You could have used Multi instead of Multiple to make the name shorter without hurting legibility but no biggie.

Reply:
Done!